delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java

```diff
@@ -182,8 +186,53 @@ public Builder setRowId(String rowId) {
     }
 
     public DMLEvent build() {
-      return new DMLEvent(offset, database, new DMLOperation(table, operationType, ingestTimestampMillis), row,
-                          previousRow, transactionId, ingestTimestampMillis, sourceTimestampMillis, isSnapshot, rowId);
+      int sizeInBytes = (operationType == DMLOperation.Type.INSERT ||
```

Comment:
should we calculate the size of Delete from previousRow ?
it seems not reasonable for size 0 for delete operation

Reply:
for example, if there are all delete operations, our metrics will look like we didn't process any data

Reply:
It's the amount of bytes we write to the BigQuery. Since for DELETE we don't write anything @bdmogal suggested to count it as 0.